examples/deck.gl/3d-tiles/3d-tile-layer/tile-3d-layer.js

```diff
@@ -89,15 +89,15 @@ export default class Tile3DLayer extends CompositeLayer {
     return await this._loadTileset(url, {headers}, ionMetadata);
   }
 
-  _updateTileset(tileset3d) {
+  async _updateTileset(tileset3d) {
     const {timeline, viewport} = this.context;
     if (!timeline || !viewport || !tileset3d) {
       return;
     }
 
     // use Date.now() as frame identifier for now and later used to filter layers for rendering
     const frameState = getFrameState(viewport, Date.now());
-    tileset3d.update(frameState);
+    await tileset3d.update(frameState);
```

Comment:
I am really not sure about this being async... I think we need a method that returns all the loaded tiles that fits into the current view, we shouldn't wait until everything has loaded to render what we already have.